src/components/jsmd-editor.jsx

```diff
@@ -169,10 +184,13 @@ function mapStateToProps(state) {
   if (state.wrapEditors === true) {
     editorOptions.lineWrapping = true;
   }
-
+  const { editorCursorLine, editorCursorChar, editorCursorForceUpdate } = state;
   return {
     content: state.jsmd,
-    editorOptions
+    editorOptions,
+    editorCursorLine,
+    editorCursorChar,
+    editorCursorForceUpdate
```

Comment:
Is there any way you could add some tests for the jsmd editor here? Perhaps experiment with modifying the state and that it updates as expected? That seems much more likely to break that the redux action/store stuff that is currently being tested.

Reply:
good point